build/pipelines/templates/build-console-steps.yml

```diff
@@ -101,3 +101,11 @@ steps:
     PathtoPublish: '$(Build.ArtifactStagingDirectory)/appx'
     ArtifactName: 'appx-$(BuildConfiguration)'
   condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+  
+- task: powershell@2
+  displayName: 'Source Index PDBs'
+  inputs:
+    targetType: filePath
+    filePath: build\scripts\Index-Pdbs.ps1
+    arguments: -SearchDir '${{ parameters.buildOutputDir }}\$(buildConfiguration)' -SourceRoot '$(Build.SourcesDirectory)' -recursive -Verbose -CommitId $(Build.SourceVersion)

```

Comment:
>${{ parameters.buildOutputDir }} [](start = 27, length = 32)

This buildOutputDir doesn't appear to be a parameter for this template

Reply:
Ah, yeah. Good call.